### DISCUSSION TITLE
Revoke blockage of risky action.

### DISCUSSION THREAD

**Discussion by dazu89** [simeononsecurity/Windows-Optimize-Harden-Debloat#50] [Q&A] [answered]
Windows Security's _Attack surface reduction_ blocks the execution of executables given that they are _untrusted and unsigned processes that ran from USB_. Is there an easier way to mark the flagging as false-positive or creating an exception to the rule, other than than adding the executable to  Exclude files and paths from Attack surface reduction rules via Group Policy Editor as described [here](https://learn.microsoft.com/en-gb/microsoft-365/security/defender-endpoint/enable-attack-surface-reduction?view=o365-worldwide#group-policy)?

**Comment by simeononsecurity** [comment, marked as answer]
Ideally you shouldn't be running any untrusted or unsigned software nor should you really be launching any software from a USB device. However this can be needed in some development environments.
You can set the attack surface reduction to audit only if it gives you issues, create a rule exception, or copy the file from the USB to the system and try to run it from there.
